<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationRulesWidget</class>
 <widget class="QWidget" name="NotificationRulesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTreeView" name="RulesTree_"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="AddRule_">
         <property name="text">
          <string>New</string>
         </property>
         <property name="icon">
          <iconset theme="document-new"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddFromMissed_">
         <property name="text">
          <string>New from missed...</string>
         </property>
         <property name="icon">
          <iconset theme="document-new-from-template"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveRuleUp_">
         <property name="text">
          <string>Move up</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-up"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="MoveRuleDown_">
         <property name="text">
          <string>Move down</string>
         </property>
         <property name="icon">
          <iconset theme="arrow-down"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="RemoveRule_">
         <property name="text">
          <string>Remove</string>
         </property>
         <property name="icon">
          <iconset theme="list-remove"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DefaultRules_">
         <property name="text">
          <string>Default</string>
         </property>
         <property name="icon">
          <iconset theme="document-revert"/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabWidgetPage1">
      <attribute name="title">
       <string>Rule</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Name:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="RuleName_"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Event category:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="EventCat_"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Event type:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTreeWidget" name="EventTypes_">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Notify via:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QCheckBox" name="NotifySysTray_">
                <property name="text">
                 <string>System tray</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="NotifyVisual_">
                <property name="text">
                 <string>Visual</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="NotifyAudio_">
                <property name="text">
                 <string>Audio</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="NotifyCmd_">
                <property name="text">
                 <string>Command</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="NotifyUrgent_">
                <property name="text">
                 <string>Draw attention to the window</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="NotifySystemDependent_">
                <property name="text">
                 <string>System-dependent</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="RuleSingleShot_">
             <property name="text">
              <string>Single shot</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Fields</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Matches:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="MatchesTree_"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="AddMatch_">
           <property name="text">
            <string>Add...</string>
           </property>
           <property name="icon">
            <iconset theme="list-add"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ModifyMatch_">
           <property name="text">
            <string>Modify...</string>
           </property>
           <property name="icon">
            <iconset theme="document-edit"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveMatch_">
           <property name="text">
            <string>Remove</string>
           </property>
           <property name="icon">
            <iconset theme="list-remove"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageAudio_">
      <attribute name="title">
       <string>Audio</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>File:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QComboBox" name="AudioFile_">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BrowseAudioFile_">
           <property name="text">
            <string>Browse...</string>
           </property>
           <property name="icon">
            <iconset theme="document-open"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="TestAudio_">
           <property name="text">
            <string>Test</string>
           </property>
           <property name="icon">
            <iconset theme="media-playback-start"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="PageCommand_">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Command</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Command:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="CommandLineEdit_"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Arguments:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTreeWidget" name="CommandArgsTree_">
           <column>
            <property name="text">
             <string>Argument</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="AddArgument_">
             <property name="text">
              <string>Add...</string>
             </property>
             <property name="icon">
              <iconset theme="list-add"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ModifyArgument_">
             <property name="text">
              <string>Modify...</string>
             </property>
             <property name="icon">
              <iconset theme="document-edit"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RemoveArgument_">
             <property name="text">
              <string>Remove</string>
             </property>
             <property name="icon">
              <iconset theme="list-remove"/>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Other</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Associated color:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="LeechCraft::Util::ColorButton" name="ColorButton_">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="UpdateRule_">
     <property name="text">
      <string>Save rule</string>
     </property>
     <property name="icon">
      <iconset theme="document-save"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LeechCraft::Util::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">util/gui/colorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
